common/build/eslint-config-fluid/minimal.js

```diff
@@ -142,10 +142,28 @@ module.exports = {
         "@typescript-eslint/comma-spacing": "error",
         "@typescript-eslint/func-call-spacing": "error",
         "@typescript-eslint/keyword-spacing": "error",
+        "@typescript-eslint/member-delimiter-style": [
+            "error",
+            {
+                "multiline": {
+                    "delimiter": "semi",
+                    "requireLast": true
+                },
+                "singleline": {
+                    "delimiter": "semi",
+                    "requireLast": true
```

Comment:
The default for single-line is false, which seems inconsistent to me, so I changed it. The tree projects, which currently use prettier, also omit trailing semicolons in single-line cases. What say you all?

Reply:
No strong opinion from me, my default is to prefer minimal change to the repo unless/until someone has a strong opinion.